clients/osv/src/main/kotlin/Model.kt

```diff
@@ -75,6 +75,25 @@ data class Credit(
     val contact: List<String> = emptyList()
```

Comment:
Commit message nit: In the first line of the body "the changing" -> "then changing".

I was a bit irritated that the whole body of the commit message is a reasoning why no enum class is used, while I was expecting some background about the new class. To make clearer that you mainly refer to this implementation detail, maybe start with something like: "Note that for this purpose no enum is used because ..." ?